dali/base/dadfs.cpp

```diff
@@ -3566,23 +3566,33 @@ class CDistributedFile: public CDistributedFileBase<IDistributedFile>
 
         offset_t maxPartSz = 0, minPartSz = (offset_t)-1, totalPartSz = 0;
 
-        maxSkewPart = 0;
-        minSkewPart = 0;
-        for (unsigned p=0; p<np; p++)
+        try
         {
-            IDistributedFilePart &part = queryPart(p);
-            offset_t size = part.getFileSize(true, false);
-            if (size > maxPartSz)
-            {
-                maxPartSz = size;
-                maxSkewPart = p;
-            }
-            if (size < minPartSz)
+            maxSkewPart = 0;
+            minSkewPart = 0;
```

Comment:
Minor, I might also set:
minSkew = maxSkew = 0;
here so they are set if an exception occurred, but I don't think these args are ever used if this routine returns false so it doesn't really matter.